README.md

```diff
@@ -248,8 +248,15 @@ QAndQDotBounds waits for a biorbd model and returns a structure with the minimal
 How convenient!
 ```python
 x_bounds = QAndQDotBounds(biorbd_model)
-```
-Then, override the first and last column to be 0, that is the sideways and rotation to be null for both the position and the velocities
+
+
+The first element of x_bounds is the degree of freedom declared in order in the bio.Mod file.
+In this case, the degree of freedom with index 0 is translation y, and the index 1 refers to rotation x. 
+Finally, the index 2 and 3 are respectively the velocity of translation y and rotation x 
+
+The time is discretized in nodes wich is the second element declared in x_bounds
+
+We want the first and last column(which is equivalent to nodes 0 and -1) to be 0, that is the translations and rotations to be null for both the position and so the velocities, wich finally means that all the degrees of freedom are null.
```

Comment:
which*

degrees of freedom or *q* do not refer to both position and velocities, but only to position. So this final part (from which to the end) is misleading.